### DISCUSSION TITLE
Add GCP Deployment Docs

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#4893] [Feature Requests]
We have AWS, Heroku, etc... but it seems that GCP is increasingly requested. Let's get an initial guide for Google Cloud Platform added here:

https://docs.directus.io/getting-started/installation/#deployment-guides

**Comment by keesvanbemmel** [comment]
You can assign this to me if you want, more than happy to write up what we have in production

**Reply by rijkvanzanten** [reply]
Yes please!

**Reply by keesvanbemmel** [reply]
I'll try to do so next week!

**Reply by keesvanbemmel** [reply]
Just a quick FYI and update @rijkvanzanten @benhaynes I have not forgotten about this, just takes a while longer due to holidays etc :) 
<img width="1904" alt="CleanShot 2021-12-19 at 11 19 25@2x" src="https://user-images.githubusercontent.com/4633876/146671534-9d751e2d-0b53-4bd2-8ba5-daac45193722.png">

**Reply by rijkvanzanten** [reply]
No worries! Fijne kerstdagen en alvast een gelukkig nieuwjaar 🙂

**Reply by keesvanbemmel** [reply]
https://github.com/directus/directus/pull/10936 there you go!

**Comment by keesvanbemmel** [comment]
For anyone landing in this thread. Google Cloud documentation is now available: [GCP guide](https://docs.directus.io/getting-started/installation/gcp/)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by emilienbidet** [comment]
Hello, 

I would like to self host an instance of Directus on GCP but the guide is not in the new docs.

This was the perfect doc with :
- Cloud Storage
- Cloud SQL
- Cloud Run

@keesvanbemmel do you still have the documentation about GCP ?

Thank you very much
Emilien

**Reply by Polluxs** [reply]
For anyone still interested:

[web archive](https://web.archive.org/web/20220118100937/https://docs.directus.io/getting-started/installation/gcp/)
